apps/coscrad-frontend/src/configurable-front-matter/data/content.config.SAMPLE.ts

```diff
@@ -124,4 +124,7 @@ export const contentConfig: ConfigurableContent = {
     },
     resourceIndexLabel: 'Available Resources',
     defaultLanguageCode: LanguageCode.Chilcotin,
+    phoneNumber: '123456789',
```

Comment:
You should make it very clear here whether the formatting is added up stream or whether this should be formatted here as desired (e.g., `1-(234) 567-8912`.